crates/sovereign/sovereign-chain-components/src/sovereign/impls/sovereign_to_cosmos/connection/connection_handshake_payload.rs

```diff
@@ -88,18 +87,12 @@ where
         let rollup_connection_end =
             query_connection_end(chain.rollup(), connection_id, &rollup_height).await;
 
-        let proof_try = CommitmentProofBytes::try_from(rollup_connection_end.proof)
-            .map_err(Chain::raise_error)?;
-
         let rollup_client_state =
             query_client_state(chain.rollup(), client_id, &rollup_height).await;
```

Comment:
I have added the interfaces for queries with proofs, such as `CanQueryClientStateWithProofs`. So we can refactor the implementations to use the new interfaces.